silverback/cluster/client.py

```diff
@@ -452,6 +452,27 @@ def create_workspace(
         self.workspaces.update({new_workspace.slug: new_workspace})  # NOTE: Update cache
         return new_workspace
 
+    def remove_workspace(self, workspace_slug):
+        workspace_id = self.workspaces[workspace_slug].id
+        response = self.delete(f"/workspaces/{str(workspace_id)}")
```

Comment:
`str(...)` is automatically called on anything inside format strings
```suggestion
        response = self.delete(f"/workspaces/{workspace_id}")
```